### ISSUE TITLE
Parsing error when folder names have "@"

### ISSUE THREAD

**Issue by a-ignatev** [fan-tom/intellij-codeowners#108]
When the path has a folder name with "@" the syntax highlight is failing with the message: 
```
CodeownersTokenType.CRLF or CodeownersTokenType.VALUE expected, got '@toto/team'
```

For example, if you have this line:
```
packages/@foo/bar/file.ts @toto/team @toto/other-team
```

**Comment by dulishkovych** [comment]
Yes please, my team struggles with this bug as well!

**Comment by fan-tom** [comment]
Closed in #104
